.github/workflows/publish-ignition-preview.yml

```diff
@@ -0,0 +1,85 @@
+name: Publish Ignition Preview
```

Comment:
I think we should keep all publish Vercel logic on the `publish-ignition.yml`, create a new `preview.yml` workflow just to manually trigger the deployment passing the required params